tests/auth/register.spec.js

```diff
@@ -0,0 +1,59 @@
+const { test, expect } = require('@playwright/test');
+const { faker } = require('@faker-js/faker');
+const userEmail = faker.internet.email()
+
+test.describe.only('Login page', () => {
+test('Register form', async ({page})=> {
+ await page.goto('/')
+ const signInButton= page.locator('button', {hasText: 'Sign In'})
+ const loginPopup= page.locator('div.modal-dialog-centered')
+ const registrationButton= loginPopup.locator('button', {hasText: 'Registration'})
+ const registrationPopup= page.locator('.modal-dialog.modal-dialog-centered')
+ const registrationTitle= registrationPopup.locator('.modal-title')
+ const nameInput= registrationPopup.locator('#signupName')
+ const lastNameInput= registrationPopup.locator('#signupLastName')
+ const emailInput= registrationPopup.locator('#signupEmail')
+ const passwordInput= registrationPopup.locator('#signupPassword')
+ const repeatPasswordInput= registrationPopup.locator('#signupRepeatPassword')
+ const registerButton= registrationPopup.locator('button', {hasText: 'Register'})
+ const expectedValidationMessages = [
+    'Name is invalidName has to be from 2 to 20 characters long',
+    'Last name is invalidLast name has to be from 2 to 20 characters long',
+    'Email is incorrect',
+    'Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter',
+  ];
+
+
+ 
+ await signInButton.click()
+ await expect(loginPopup).toBeVisible()
+ await registrationButton.click()
```

Comment:
open popup by click on this element
![image](https://github.com/TaniaBystrova/playwright/assets/53534240/ee1e8753-4ba0-440b-8495-9fc3688581a4)

Reply:
+